### PR TITLE
docs(radial-gauge)

### PR DESCRIPTION
**Pull request by ntacheva**
## New Component Checklist (delete for other types of PRs)

- [x] - Overview article
    - [x] General information on the purpose of the component.
    - [x] Instruction on how to add the component to a razor component.
    - [x] Basic configuration of the component - code snippet with a scenario that is both basic, and meaningful.
    - [x] Provide information on the component's reference.
    - [x] List the available features for the component and/or provide links (slugs) for the articles that explain them in details. For simpler components listing parameters might be enough, for more complex ones sections with a sentence or two that links corresponding articles might be more suitable.
    - [x] Maybe some Example(s) - very common scenarios, notes, typical use case, etc. Not mandatory, depends on the component.

- [x] - Feature/component specific articles - those articles explain the component specific features, which do not suit in any of the before-mentioned articles. Examples of such articles are the [Navigator](https://docs.telerik.com/blazor-ui/components/stockchart/navigator) for the Stock Chart, the [Indeterminate](https://docs.telerik.com/blazor-ui/components/checkbox/indeterminate-state) state for the CheckBox and others.

### CODE REVIEW

**Review by marin-bratanov — APPROVED** [review]

File: components/gauge-radial/overview.md [hunk 1]
```diff
@@ -1,147 +1,110 @@
...
-        ContainerHeight = "500px";
-        ContainerWidth = "800px";
+        Height = "400px";
 
-        //give time to the framework and browser to resize the actual DOM so the chart can use the expected size
-        await Task.Delay(20);
-
-        //redraw the chart
-        theChart.Refresh();
+        RadialGaugeRef.Refresh();
```
[the remark this answers is not in the document]
Comment (svdimitr) [reply components/gauge-radial/overview.md:99]: It does not, It needs the delay, I have logged an issue https://github.com/telerik/blazor/issues/2255 after syncing with Stamo

File: components/gauge-radial/overview.md [hunk 2]
```diff
@@ -1,147 +1,110 @@
...
 
-<TelerikChart @ref="myChartRef">
-</TelerikChart>
+* `Class` - renders a custom CSS class on the topmost wrapping element of the component. You can use that class to reposition the component on the page.
 
-@code {
-	Telerik.Blazor.Components.TelerikChart myChartRef;
-}
-````
+* Scale - The scale of the radial gauge renders the values of the [pointers]({%slug radial-gauge-pointers%}), different [ranges]({%slug radial-gauge-ranges%}) and [labels]({%slug radial-gauge-labels%}). See the [Scale]({%slug radial-gauge-scale%}) article for more information on how to customize the scale of the component.
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/gauge-radial/overview.md:64]: Done

File: components/gauge-radial/scale.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    <RadialGaugePointers>
+
+        <RadialGaugePointer Value="30">
+        </RadialGaugePointer>
+
+    </RadialGaugePointers>
+
+    <RadialGaugeScales>
+
+        <RadialGaugeScale StartAngle="30" EndAngle="150">
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/gauge-radial/scale.md:122]: Done

File: components/gauge-radial/overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/gauge-radial/overview.md:99]: Done

**Review by marin-bratanov — APPROVED** [review]

**Pull request merged**